apps/server/lib/google-drive/src/resources/apps.ts

```
@@ -0,0 +1,367 @@
+export namespace AppListParams {
+	export interface _ {
+		/**
+		 * V1 error format.
+		 */
+		xgafv?: "1" | "2";
+	}
+}
+
+export declare namespace Apps {
+	export {
+		type App as App,
```

Biome (1.9.4)

[error] Useless rename.

Safe fix: Remove the renaming.


(lint/complexity/noUselessRename)